lib/Target/AVR/AVRInstrInfo.cpp

```diff
@@ -45,6 +45,30 @@ extern bool RA_InSpillerCode;
 AVRInstrInfo::AVRInstrInfo()
     : AVRGenInstrInfo(AVR::ADJCALLSTACKDOWN, AVR::ADJCALLSTACKUP), RI() {}
 
+// FIXME: BEFORE MERGING: Is there a cleaner tablegen method of doing this?
```

Comment:
This worked well, thanks! The harder question: is this a good idea? Does it make sense to try to copy from a 16-bit register to an 8-bit one? What about the other way around?